src/pages/404.tsx

```diff
@@ -8,7 +8,7 @@ const FourOhFour: FunctionComponent = (): JSX.Element => {
   const pageHeading = "Page not found";
 
   return (
-    <Layout title={pageHeading} pageHasErrors={false}>
+    <Layout title={pageHeading} pageHasErrors={false} showCookieBanner={false}>
```

Comment:
See description for PR. Have removed as the 404 page is statically rendered => we cannot determine if the user has accepted the cookie policy. Decided to not display the cookie banner on the 404 page would be the easiest solution to this.

Reply:
Agreed